core/src/components/datetime/datetime.scss

```diff
@@ -25,19 +25,6 @@
   overflow: hidden;
 }
 
-/**
- * When using the wheel picker to switch
- * between months, sometimes the allowed
- * dates may be filtered. As a result, it
- * is possible to get a layout shift as
- * the picker column will shrink to fit the
- * widest item in the column. Setting a minimum
- * width avoids this layout shifting.
- */
-ion-picker-column {
-  min-width: 26px;
-}
```

Comment:
I had to remove this from datetime and put it in picker-column so it can be moved to `.picker-opts`, which is in the shadow DOM.